Bottom: 0e5daf8cdc3e52d373e98398cbd8398b7164c046
Top:    ff41261693dfca15a19bb7c920d5072c50491376
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 20:05:51 +0000

Refresh of odds-and-sods

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 670c1df..546bf45 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1504,7 +1504,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   /* DEBUG */
   poly_M_Copy0 (&PCB->Data->outline, info.poly);
   PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
 
diff --git a/src/polygon.h b/src/polygon.h
index aabc096..1343d03 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
